sql/moz-fx-data-shared-prod/mozilla_org_derived/gclid_conversions_v1/schema.yaml

```diff
@@ -0,0 +1,21 @@
+fields:
+  - name: activity_date
+    mode: NULLABLE
+    type: DATE
+    description: "The date that the conversion action took place."
+  - name: gclid
+    mode: NULLABLE
+    type: STRING
+    description: "A Google Click ID, which uniquely represent an ad impression for Google ads."
+  - name: did_search
+    mode: NULLABLE
+    type: BOOLEAN
+    description: "Whether the GCLID did a search on this day."
```

Comment:
super nit: I'd write whether the client that came from the GCLID did the thing. It's subtle but the GCLID is an impression but not a "user"